_includes/head.html

```diff
@@ -6,38 +6,38 @@
   
   <!-- Opengraph card (WhatsApp & others)  -->
   <meta property="og:title" content="Zeppelin" />
   <meta property="og:description" content="Zeppelin: infrastructure for decentralized systems" />
-  <meta property="og:url" content="{{ "./" | absolute_url}}" />
+  <meta property="og:url" content="{{ "/" | absolute_url}}" />
   <meta property="og:site_name" content="Zeppelin" />
-  <meta property="og:image" content="{{ "./images/card.jpg" | absolute_url }}"/>
-  <meta property="og:image:secure_url" content="{{ "./images/card.jpg" | absolute_url }}"/>
+  <meta property="og:image" content="{{ "/images/card.jpg" | absolute_url }}"/>
+  <meta property="og:image:secure_url" content="{{ "/images/card.jpg" | absolute_url }}"/>
   <meta property="og:image:width" content="300" />
   <meta property="og:image:height" content="300" />
   
   <meta name="description" content="Zeppelin: infrastructure for decentralized systems">
 
   <!-- favicon -->
-  <link rel="apple-touch-icon" sizes="57x57" href="{{ "/images/favicon/apple-icon-57x57.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="60x60" href="{{ "/images/favicon/apple-icon-60x60.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="72x72" href="{{ "/images/favicon/apple-icon-72x72.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="76x76" href="{{ "/images/favicon/apple-icon-76x76.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="114x114" href="{{ "/images/favicon/apple-icon-114x114.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="120x120" href="{{ "/images/favicon/apple-icon-120x120.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="144x144" href="{{ "/images/favicon/apple-icon-144x144.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="152x152" href="{{ "/images/favicon/apple-icon-152x152.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="180x180" href="{{ "/images/favicon/apple-icon-180x180.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="192x192"  href="{{ "/images/favicon/android-icon-192x192.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="32x32" href="{{ "/images/favicon/favicon-32x32.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="96x96" href="{{ "/images/favicon/favicon-96x96.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="16x16" href="{{ "/images/favicon/favicon-16x16.png" | absolute_url }}"/>
-  <link rel="manifest" href="{{ "/images/favicon/manifest.json" | absolute_url }}"/>
+  <link rel="apple-touch-icon" sizes="57x57" href="/images/favicon/apple-icon-57x57.png"/>
+  <link rel="apple-touch-icon" sizes="60x60" href="/images/favicon/apple-icon-60x60.png"/>
+  <link rel="apple-touch-icon" sizes="72x72" href="/images/favicon/apple-icon-72x72.png"/>
+  <link rel="apple-touch-icon" sizes="76x76" href="/images/favicon/apple-icon-76x76.png/">
```

Comment:
Accidentally switched characters.
```suggestion
  <link rel="apple-touch-icon" sizes="76x76" href="/images/favicon/apple-icon-76x76.png"/>
```